llvm/lib/Transforms/HelloNew/HelloWorld.h

```diff
@@ -0,0 +1,15 @@
+#ifndef LLVM_TRANSFORMS_HELLONEW_HELLOWORLD_H
```

Comment:
Why is this file committed here? Isn't it already presented in the same form in llvm/include/llvm/Transforms/HelloNew/HelloWorld.h -- I guess so, so why have you copied over?